nomad/grapl-core.nomad

```diff
@@ -617,21 +634,23 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image        = "${var.container_registry}grapl/graph-merger:${var.graph_merger_tag}"
-        network_mode = "grapl-network"
+        image = "${var.container_registry}grapl/${var.container_repo}graph-merger:${var.graph_merger_tag}"
+      }
+
+      template {
+        data        = local.local_vars
```

Comment:
+1

Reply:
I'm changing it to conditionally_defined_env_vars, which I think is a bit better since we're likely going to have env vars defined only for AWS as well